Tasks/ServiceFabricDeployV1/Create-DiffPackage.psm1

```diff
@@ -105,8 +106,12 @@ function Create-DiffPackage
             Copy-DiffPackage -clusterPackages $clusterServiceManifest.ConfigPackage -localPackages $localServiceManifest.ConfigPackage -localParentPkgPath $localServicePkgPath -diffParentPkgPath $diffServicePkgPath
             Copy-DiffPackage -clusterPackages $clusterServiceManifest.DataPackage -localPackages $localServiceManifest.DataPackage -localParentPkgPath $localServicePkgPath -diffParentPkgPath $diffServicePkgPath
 
-            Write-Host (Get-VstsLocString -Key DIFFPKG_CopyingToDiffPackge -ArgumentList @($localServiceManifestPath, (Join-Path $diffServicePkgPath $serviceManifestName)))
-            Copy-Item -LiteralPath $localServiceManifestPath (Join-Path $diffServicePkgPath $serviceManifestName) -Force
+            Write-Host (Get-VstsLocString -Key DIFFPKG_CopyingToDiffPackge -ArgumentList @($localServiceManifestPath, $diffPkgServiceManifestPath))
+            if (!(Test-Path -PathType Container -LiteralPath $diffServicePkgPath))
+            {
+                $diffServicePkgPath = New-Item -ItemType Directory -Path $diffServicePkgPath -Force
```

Comment:
Why not keep it simple and always create destination dir first before copying diff packages?

Reply:
@bishal-pdMSFT If a service is not changed and not copied into the diff package at all, why do we create an empty folder for the service?

Reply:
But we have already checked that service manifest version is greater than in cluster. line 98. So, over here you will always need to create folder and copy manifest file. Am I missing something?

Reply:
@bishal-pdMSFT You are right. We can create folder after the check